department-of-veterans-affairs/abd-vro
# Remove Configurations for MAS Integration and Helm Chart for RRD

## What was the problem?

Some lingering config options related to MAS integration from back in the RRD days

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]


## How to test this PR
- Deploy to escalating LHDI environments

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
